### DISCUSSION TITLE
входящее termux/sshd соединение когда ByeDPIAndroid/VPN включен

### DISCUSSION THREAD

**Discussion by BeragesFrares** [dovecoteescapee/ByeDPIAndroid#144] [Q&A]
У меня настроен termux/sshd так, что SCP соединение на порту 8022 работает как файловая сессия, то есть в termux-е толкаю sshd, после этого из FAR Manager-а соединяюсь по SCP. Но это не работает одновременно с ByeDPIAndroid/VPN . То есть либо когда vpn включен, то не соединяется, либо разрывает файловую сессию, когда я жму connect в ByeDPIAndroid. Пока это означает, что я должен помнить, что они не могут одновременно работать. Но не думаю, что что-то тут принципиально, что препятствовать должно. Мне нужно изменить какие-то настройки? issue завести?

Спасибо

**Comment by krlvm** [comment]
#132 должно помочь
